### PR TITLE
Fix snippet on android

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers Fixes snippet cutoff problem on android

### CODE REVIEW

File: shared/chat/inbox/row/bottom-line.js [hunk 1]
```diff
@@ -120,15 +120,24 @@ const noWrapStyle = {
...
+  fontSize: 13,
+  lineHeight: lineHeight(17),
+  marginTop: 2,
+  paddingRight: 30,
+}
+
+const mobileBottomLine = isAndroid
+  ? {
+      ...mobileBottomLineCommon,
+      lineHeight: undefined,
```
Comment (MarcoPolo) [inline shared/chat/inbox/row/bottom-line.js]: lineHeight isn't useful for 1 line snippets so we can do this safely

**Comment by chrisnojima** [comment]
But why is this cut off? Isn't the emoji within the line height?

**Comment by MarcoPolo** [comment]
The text was cutoff not just emojis. I think it was a combination of two things:

1. lineHeight behaves differently on ios/android (https://github.com/facebook/react-native/issues/10712)

2. Text can go outside it's container on ios, but gets cutoff on android. This is why the {min,max}Height had to be changed.

**Comment by chrisnojima** [comment]
i wonder if we should set overflow: hidden on ios so we can see this type of problem there also (can be off for prod build) just thinking out load

**Pull request merged**